graphql/src/schema/ast.rs

```diff
@@ -411,6 +412,23 @@ pub fn get_input_object_definitions(schema: &Document) -> Vec<InputObjectType> {
         .collect()
 }
 
+/// Determines if the field is an imported type
+pub fn is_field_imported(
+    subgraph_id: &SubgraphDeploymentId,
+    schema: &Document,
+    field_definition: &Field,
+) -> bool {
+    let field_type_name = get_field_name(&field_definition.field_type);
+    Schema::new(subgraph_id.clone(), schema.clone())
+        .imported_types()
```

Comment:
Could `imported_types` also go into the extension trait? That would make it unnecessary to construct a `Schema` just to extract imported types.